sunpy/net/vso/attrs.py

```diff
@@ -400,7 +400,12 @@ class Quicklook(_VSOSimpleAttr):
     Quicklook products are *not* searched by default.   Reference:
     documentation in SSWIDL routine vso_search.pro.
     """
-    pass
+    def __init__(self,value):
+        super(Quicklook,self).__init__(value)
+        if(self.value == True):
```

Comment:
Well the orignal issue was that if you use True instead of 1 it breaks.

But yes, if i remove the ``==`` it works either way. Plus it will be faster.